src/disclosure-manager/withDisclosureManager.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import DisclosureManagerContext from './DisclosureManagerContext';
```

Comment:
I'm sure the answer is the same, but can we reuse the provided `withDisclosureManager` implementation? This is one-to-one

Reply:
Since we will be deprecating the existing packages, this is getting moved over wholesale.